### ISSUE TITLE
Help Desk emails has "From: Dev Ops <extsd@oicr.on.ca>"

### ISSUE THREAD

**Issue by christinayung** [icgc-argo/roadmap#476]
Please change "From: Dev Ops" to "From: ICGC ARGO Helpdesk"

![image.png](https://images.zenhubusercontent.com/5bbe9c22307bc60241e3aa93/dd3baff3-1baf-4518-8111-30c12b8252d4)

Also got a second email to welcome me.  Should modify the message so it doesn't say "Dev Ops has invited you ..."

![image.png](https://images.zenhubusercontent.com/5bbe9c22307bc60241e3aa93/930d7e71-d627-40d9-bf9c-6f5b8274c41c)

IT Tickets: 
https://servicedesk.oicr.on.ca/servicedesk/customer/portal/2/IT-29315
https://servicedesk.oicr.on.ca/servicedesk/customer/portal/2/IT-27703

**Comment by rosibaj** [comment]
Idea: Change the name of the Integrations User in Jira

**Comment by wajiha-oicr** [comment]
I have changed the username from "Dev Ops" to "ICGC ARGO Helpdesk". I would like to submit a request to test and see the new email, but the contact form is only accessible through prod (I can't access). 

Also, the ability to customize the invitation email only seems to be possible in the [cloud version ](https://confluence.atlassian.com/servicedeskcloud/blog/2018/07/customize-your-invitation-emails-to-customers)of the service helpdesk, which we don't have. I poked through the customer notifications settings section and can confirm there isn't an option to change that. So I think it will always end up saying "<integration user's name> has invited you...". Meaning the new message will probably be:

"ICGC ARGO Helpdesk has invited you to the ICGC ARGO Helpdesk portal!" A tad redundant but less confusing than devops.
